apps/omg/lib/omg/state.ex

```diff
@@ -159,6 +159,20 @@ defmodule OMG.State do
 
   @doc """
   see `exec/2`
+  see `deposit/1`
+  see `exit_utxos/1`
+    Flow:
+      - translates the triggers to UTXO positions digestible by the UTXO set
+      - exits the UTXOs from the ledger if they exists, reports invalidity wherever they don't
+      - returns the `db_updates` to be applied by the caller
+  see `utxo_exists/1`
```

Comment:
Something's off with this doc.

Reply:
can you be more precise?

Reply:
The comment that `exec` works exactly like `handle_cast(:form_block)` seems strange. Plus `see something` comments are scattered all over the place.